### PR TITLE
Add metadata support for VDC networks

### PR DESCRIPTION
**Pull request by adambarreiro**
## Description

Adds metadata support for VDC networks with CRUD functions.

### CODE REVIEW

**Review by lvirbalas — APPROVED** [review]
Only one polishing nit and LGTM!

**Review by Didainius — APPROVED** [review]
LGTM

**Review by mikeletux — APPROVED** [review]
LGTM! Approved! Thanks @adambarreiro , great work!

File: .changes/v2.15.0/442-improvements.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Added support to set, get and delete metadata to VDC Networks with the methods
+  `OrgVDCNetwork.AddMetadataEntry`, `OrgVDCNetwork.AddMetadataEntryAsync`, `OrgVDCNetwork.GetMetadata`,
+  `OrgVDCNetwork.DeleteMetadataEntry` and `OrgVDCNetwork.DeleteMetadataEntryAsync`. [GH-442]
```
[the remark this answers is not in the document]
Comment (adambarreiro) [reply .changes/v2.15.0/442-improvements.md]: Done, thanks!

**Review by vbauzysvmware — APPROVED** [review]
LGTM
It's worth mentioning, that `AddMetadataEntry` signature changed.

**Pull request merged**